### DISCUSSION TITLE
Nested Checkboxes

### DISCUSSION THREAD

**Discussion by urbanisierung** [carbon-design-system/carbon#11916] [Help] [answered]
[Carbon Docs ](https://carbondesignsystem.com/components/checkbox/usage/#nesting) describe nested checkboxes with parent and child elements:

![image](https://user-images.githubusercontent.com/1721647/183034255-d17418b8-9c02-4c8e-945b-18a53e1f2fa1.png)

I couldn't find any code docs about that. Is there a ready to use component available? Do you have an example for that?

Thanks!

**Comment by urbanisierung** [comment]
Any suggestions?

**Comment by tay1orjones** [comment, marked as answer]
Nested checkboxes don't have a code implementation out of the box right now. The usage guidance from the website shows a _pattern_ of how this could/should work.

If you're interested in contributing, this could be a great enhancement for the checkbox to automatically apply proper padding/margin to nested checkboxes when composed inside one another.
